crypto/keys/mintkey.go

```diff
@@ -16,6 +16,21 @@ const (
 	blockTypePubKey  = "TENDERMINT PUBLIC KEY"
 )
 
+// Make bcrypt security parameter var, so it can be changed within the lcd test
+// Making the bcrypt security parameter a var shouldn't be a security issue:
+// One can't verify an invalid key by maliciously changing the bcrypt
+// parameter during a runtime vulnerability. The main security
+// threat this then exposes would be something that changes this during
+// runtime before the user creates their key. This vulnerability must
+// succeed to update this to that same value before every subsequent call
+// to gaiacli keys in future startups / or the attacker must get access
+// to the filesystem. However, with a similar threat model (changing
+// variables in runtime), one can cause the user to sign a different tx
+// than what they see, which is a significantly cheaper attack then breaking
+// a bcrypt hash. (Recall that the nonce still exists to break rainbow tables)
+// TODO: Consider increasing default
```

Comment:
Put this long comment in the code, since I think its going to be easier for anyone wondering about this decision to read the code rather than to locate the relevant github issue.

Reply:
sure